src/core.ts

```diff
@@ -56,9 +56,9 @@ router.get('/version', async (ctx, _next) => {
   setKoaResponse(response, ctx.response);
 });
 
-router.get('/:did', async (ctx, _next) => {
-  // Strip away the first '/' character.
-  const didOrDidDocument = ctx.url.slice(1);
+router.get('/identifiers/:did', async (ctx, _next) => {
+  // Strip away the first '/identifiers/' string.
+  const didOrDidDocument = ctx.url.split('/identifiers/')[1];
```

Comment:
Consider making this a constant being initialized close right before its usage. Feels like there's a potential for a bug if we change the route but forget to change this?